examples/nestjs/src/main.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {INestApplication} from '@nestjs/common';
+import * as request from 'supertest';
+
+import {bootstrap} from './main';
+
+describe('App', () => {
+  let server: INestApplication;
+
+  beforeAll(async () => {
+    server = await bootstrap(3000);
```

Comment:
ideally we should find an open port - this test could flake if some other test or app has this port open